app/views/add-reduction.html

```diff
@@ -0,0 +1,164 @@
+    </div>
+
+    <div class="form-group" id="reduction_start_date">
+      <fieldset>
+        <legend>
+          <span class="form-label-bold">Reduction start date</span>
+          <span class="form-hint">For example, 31 3 1980</span>
+        </legend>
+        <div class="form-date" id="reduction_start_day">
+          <div class="form-group form-group-day">
+            <label class="form-label" for="start-day">Day</label>
+            <input class="form-control" id="start-day" name="red_start_day" type="number" min="1" pattern="[+-]?[0-9]{1,2}" max="31" onchange="calculateExpiry()">
+          </div>
+          <div class="form-group form-group-month" id="reduction_start_month">
+            <label class="form-label" for="start-month">Month</label>
+            <input class="form-control" id="start-month" name="red_start_month" type="number" min="1" pattern="[+-]?[0-9]{1,2}" max="12" onchange="calculateExpiry()">
+          </div>
+          <div class="form-group form-group-year" id="reduction_start_year">
+            <label class="form-label" for="start-year">Year</label>
+            <input class="form-control" id="start-year" name="red_start_year" type="number" min="0" pattern="[+-]?[0-9]{1,4}" max="2099" onchange="calculateExpiry()">
+          </div>
+        </div>
+      </fieldset>
+    </div>
+
+    <div class="form-group" id="reduction_end_date">
+      <fieldset>
+        <legend>
+          <span class="form-label-bold">Reduction end date</span>
+          <span class="form-hint">For example, 31 3 1980</span>
+        </legend>
+        <div class="form-date" id="reduction_end_day">
+          <div class="form-group form-group-day">
+            <label class="form-label" for="end-day">Day</label>
+            <input class="form-control" id="end-day" name="red_end_day" type="number" pattern="[+-]?[0-9]{1,2}" min="1" max="31">
+          </div>
+          <div class="form-group form-group-month" id="reduction_end_month">
+            <label class="form-label" for="end-month">Month</label>
+            <input class="form-control" id="end-month" name="red_end_month" type="number" pattern="[+-]?[0-9]{1,2}" min="1" max="12">
+          </div>
+          <div class="form-group form-group-year" id="reduction_end_year">
+            <label class="form-label" for="end-year">Year</label>
+            <input class="form-control" id="end-year" name="red_end_year" type="number" pattern="[+-]?[0-9]{1,4}" min="0" max="">
+          </div>
+        </div>
+      </fieldset>
+    </div>
+
+    <div class="form-group">
+      <label class="form-label-bold" for="textarea">Additional notes</label>
+      <textarea class="form-control form-control-3-4" name="notes" id="textarea" rows="5"></textarea>
+    </div> 
+    {% include "includes/csrf-hidden-input.html"%} 
+    <input class="button" type="submit" value="Add reduction">
+  </form>
+
+</main>
+{% endblock %} {% block body_end %}
+<script type="text/javascript">
```

Comment:
Can you break out this javascript into its own file?

Reply:
Yep, working on that now.